lib/ansible/modules/cloud/azure/azure_rm_aksversion_facts.py

```diff
@@ -18,7 +18,7 @@
 
 version_added: "2.8"
 
-short_description: Get available kubernetes versions supported by Azure Kubernetes Service.
+short_description: Get available kubernetes versions supported by Azure Kubernetes Service
```

Comment:
why remove .?

Reply:
Here is a little hint from Ansible document, short_description should not have a trailing period/full stop([Link](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation)). Thanks a lot!